tools/check_jsonnet

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+success=1
+JSONNET_FILES=$(git ls-files data/ | grep '.json\?net$' | xargs echo)
```

Comment:
we are not checking  `.libsonnet` files, which are the more interesting for QE Yam, could you please add them?

Reply:
Added libsonnet, got the CI failure: 
RuntimeError: RUNTIME ERROR: couldn't manifest function in JSON output.
	data/yam/agama/auto/lib/storage.libsonnet:(14:25)-(42:4)	object <anonymous>
	During manifestation	

OK: data/yam/agama/auto/template.libsonnet
make: *** [Makefile:[77](https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/14031182038/job/39278986230?pr=21496#step:6:78): test-jsonnet-valid] Error 1